COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp

```diff
@@ -335,6 +335,37 @@ namespace COLLADAMaya
             COLLADASW::URI sourceFileUri ( COLLADASW::URI::nativePathToUri ( currentScene ) );
             if ( sourceFileUri.getScheme ().empty () )
                 sourceFileUri.setScheme ( COLLADASW::URI::SCHEME_FILE );
+			if (sourceFileUri.scheme() == COLLADASW::URI::SCHEME_FILE && ExportOptions::sourceDataFormat() != ExportOptions::Absolute)
+			{
+				COLLADABU::URI relativeTo;
+				switch (ExportOptions::sourceDataFormat())
+				{
+					case ExportOptions::RelativeToExportedDAE:
+					{
+						relativeTo = COLLADABU::URI::nativePathToUri(getFilename());
+						break;
+					}
+					case ExportOptions::RelativeToMayaProjectWorkspace:
+					{
+						MString workspace;
+						if (MGlobal::executeCommand(MString("workspace -q -rd;"), workspace))
```

Comment:
What about Jerome's proposal to use an <extra> for that ?
I find it more elegant that adding more text to comment
Furthermore, I'd like those export parameters to be included in that <extra> too
What do you think about that @mbarnes-sb ?